daemons/dss-events-scribe/app.py

```diff
@@ -28,48 +21,43 @@
 app = domovoi.Domovoi()
 
 
-class ReplicaStatus:
-    prefix = None
-    finished_journaling = False
-    finished_updating = False
+# TODO: Make this configurable
+NUMBER_OF_EVENTS_PER_JOURNAL = 10 if "dev" == os.environ['DSS_DEPLOYMENT_STAGE'] else 1000
```

Comment:
If `dev` is different, could this potentially cause unexpected behavior when releasing to `integration`?

Reply:
I think it has to do with the amount of activity on integration, but i could be wrong.....

Reply:
I don't think so. `NUMBER_OF_EVENTS_PER_JOURNAL` is set dynamically at runtime.